### ISSUE TITLE
Getting "weird server reply"

### ISSUE THREAD

**Issue by serchavalos** [YoshiyukiKato/grpc-mock#8]
Hello!

I tried this service last weekend, when I got the mock server up and running, I tested using cURL and got a _"Weird server reply"_

```sh
curl 'http://localhost:8080/s12.iot.v1.DevicesService/ListDevices' -X POST -I
curl: (8) Weird server reply
```

Do you know if there something obvious I'm skipping?

Cheers!

**Comment by YoshiyukiKato** [comment]
@serchavalos Hi. I assume that curl does not have function to talk with gRPC server by http.

**Comment by serchavalos** [comment]
@YoshiyukiKato yes, you are right. I'm using https://github.com/grpc/grpc-web/ and thus communicates over http. my bad.

Thanks anyway

**Comment by YoshiyukiKato** [comment]
@serchavalos OK, I see. Could you show me your server app code? I never use grpc-web with grpc-mock.

**Comment by serchavalos** [comment]
@YoshiyukiKato, unfortunately, I don't have an example, but the server code is written in Go. The client in the browser which is produced by grpc-web from the proto files that these two pieces share
